src/lib/chawathe98_matcher.rs

```diff
@@ -924,167 +862,175 @@ pub fn edgecover_apply_edit_script(new_matcher_pruning : &mut MappingStoreGraph<
+                             && from.get_child_position(&from_arena).unwrap()
+                                == (get_position.unwrap())
+                             && parent_from_node_checker == parent_from_node)
+                        {
+                            let mut move_action = Move::new(from,
+                                                            parent_from_node,
+                                                            get_position.unwrap() as u16);
+                            move_action.apply(&mut from_arena);
+                            script.push(move_action);
+                            hash_from_ids_inserted.insert(from);
+                        }
                         // The if above checks if they were in the same position if they are not then move otherwise "Perform the move operation"
-                        indexes_t_f.insert(to_node.id(),from.id());
+                        indexes_t_f.insert(to_node.id(), from.id());
                     }
-                }
-                    else if edge_type == EdgeType::GLUE {
-
-                        let get_position = to_node.get_child_position(&(to_arena));
-                        // Get Parent
-                        let mut parent_from_node:NodeId<FromNodeId>;
-
-                        let parent_index_1 = to_arena[to_node].parent().unwrap().id();
-                        let parent_from_node_1 = NodeId::<FromNodeId>::new(parent_index_1);
-                        let wrapper= mapping_t_f.get(&to_arena[to_node].parent().unwrap());
-                        if wrapper.is_some(){
-                            parent_from_node = *wrapper.unwrap();
-                        } else {
-                            let parent_index = to_arena[to_node].parent().unwrap().id();
-                            parent_from_node = NodeId::<FromNodeId>::new(parent_index);
-                        }
-                        // Another way to get parent
-                        let index_parent_from_index = get_index_from_node(&indexes_t_f.clone(),parent_index_1);
-
+                } else if edge_type == EdgeType::GLUE {
+                    let get_position = to_node.get_child_position(&(to_arena));
+                    // Get Parent
+                    let mut parent_from_node: NodeId<FromNodeId>;
+                    let parent_index_1 = to_arena[to_node].parent().unwrap().id();
+                    let parent_from_node_1 = NodeId::<FromNodeId>::new(parent_index_1);
+                    let wrapper = mapping_t_f.get(&to_arena[to_node].parent().unwrap());
+                    if wrapper.is_some() {
+                        parent_from_node = *wrapper.unwrap();
+                    } else {
+                        let parent_index = to_arena[to_node].parent().unwrap().id();
+                        parent_from_node = NodeId::<FromNodeId>::new(parent_index);
+                    }
+                    // Another way to get parent
```

Comment:
What does this comment mean?

Reply:
Redundant comment. It was just me speaking to myself.